src/core/pointcloud/qgspointcloudlayerrenderer.cpp

```diff
@@ -59,7 +59,20 @@ QgsPointCloudLayerRenderer::QgsPointCloudLayerRenderer( QgsPointCloudLayer *laye
 
   mRenderer.reset( mLayer->renderer()->clone() );
   if ( !mSubIndexes.isEmpty() )
+  {
     mSubIndexExtentRenderer.reset( new QgsPointCloudExtentRenderer() );
+    mSubIndexExtentRenderer->setShowLabels( mRenderer->showLabels() );
+    if ( mRenderer->labelTextFormat() == QgsStyle::defaultStyle()->defaultTextFormat() )
```

Comment:
Not quite right :) 

The default should **only** be set in QgsPointCloudRenderer::QgsPointCloudRenderer() constructor. Then there's no text formatting related logic required here.

Reply:
Alright I understand now, I was bit confused as you mentioned layer renderer before. It _should_ be fixed now...